ra/ra_test.go

```diff
@@ -2912,58 +2879,42 @@ func TestFinalizeOrder(t *testing.T) {
 	oneDomainCSR, err := x509.CreateCertificateRequest(rand.Reader, &x509.CertificateRequest{
 		PublicKey:          testKey.PublicKey,
 		SignatureAlgorithm: x509.SHA256WithRSA,
-		DNSNames:           []string{"example.com"},
+		DNSNames:           []string{"a.com"},
 	}, testKey)
 	test.AssertNotError(t, err, "Error creating CSR with one DNS name")
 
 	twoDomainCSR, err := x509.CreateCertificateRequest(rand.Reader, &x509.CertificateRequest{
 		PublicKey:          testKey.PublicKey,
 		SignatureAlgorithm: x509.SHA256WithRSA,
-		DNSNames:           []string{"a.com", "a.org"},
+		DNSNames:           []string{"a.com", "b.com"},
 	}, testKey)
 	test.AssertNotError(t, err, "Error creating CSR with two DNS names")
 
-	threeDomainCSR, err := x509.CreateCertificateRequest(rand.Reader, &x509.CertificateRequest{
-		PublicKey:          testKey.PublicKey,
+	validCSR, err := x509.CreateCertificateRequest(rand.Reader, &x509.CertificateRequest{
+		PublicKey:          testKey.Public(),
 		SignatureAlgorithm: x509.SHA256WithRSA,
-		DNSNames:           []string{"a.com", "a.org", "b.com"},
+		DNSNames:           []string{"not-example.com", "www.not-example.com"},
 	}, testKey)
-	test.AssertNotError(t, err, "Error creating CSR with three DNS names")
+	test.AssertNotError(t, err, "Error creating CSR with authorized names")
 
-	// Create one finalized authorization for not-example.com and one finalized
-	// authorization for www.not-example.org
-	exp := ra.clk.Now().Add(365 * 24 * time.Hour)
-	authzIDA := createFinalizedAuthorization(t, sa, "not-example.com", exp, "valid", ra.clk.Now())
-	authzIDB := createFinalizedAuthorization(t, sa, "www.not-example.com", exp, "valid", ra.clk.Now())
+	// reusedKeyCSR, err := x509.CreateCertificateRequest(rand.Reader, &x509.CertificateRequest{
+	// 	// AccountKeyA is used as the keypair for the global Registration in this test file.
+	// 	PublicKey:          AccountKeyA.Key,
+	// 	SignatureAlgorithm: x509.SHA256WithRSA,
+	// 	DNSNames:           []string{"not-example.com", "www.not-example.com"},
+	// }, testKey)
+	// test.AssertNotError(t, err, "Error creating CSR with same pubkey as registration")
```

Comment:
Commented code.

Reply:
Removed! Thanks for catching this, this was a test case that I hoped to add before I realized it was already covered by `TestCertificateKeyNotEqualAccountKey`.